src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3125,6 +3197,32 @@ components:
           type: string
           example:
             "Main group"
+    HighlightInfo:
+      type: object
+      properties:
+        start:
+          type: integer
+          example: 70
+        end:
+          type: integer
+          example: 70
+        type:
+          type: string
+          example: MD
```

Comment:
Please add description:
Type of highlight.

With enum from `\Fossology\Lib\Data\Highlight`